### ISSUE TITLE
knora-base:IntervalValue: Enforce end higher than start

### ISSUE THREAD

**Issue by tobiasschweizer** [dasch-swiss/knora-api#1603]
Should Knora enforce that the end of an interval is higher than its start?

See also https://github.com/dasch-swiss/knora-ui-ng-lib/pull/26#issuecomment-590309804

**Comment by SepidehAlassi** [comment]
Yes, the end of an interval should be higher than its start. I will add a check for that in DSP-1766.
